schemas/+aod/ScanMotion.m

```diff
@@ -27,16 +27,24 @@ function makeTuples(self, key)
 		
             tuple = key;
             
-            br = getFile(acq.AodScan & key,'Motion');
-            motionData = br(:,:,:);
-            [T D P] = size(motionData);
-            mot = reshape(motionData,[T 10 20 P]);
-            mot = permute(mot, [4 2 3 1]);
-            t = br(:,'t');
-            coordinates = br.motionCoordinates;
-            
+            fn = char(fetchn(acq.AodScan & key, 'aod_scan_filename')) ; % get the filename from db
+            oldversion = aod.checkFileversion(fn) ; % check if motion data is in old format, i.e. 2 planes or in new format, i.e. a volume

```

Comment:
it's probably best to keep with the aod schema since it doesn't apply to anything else in acq. Also, if we can refactor it into the AodMotionReader class so that this isn't exported in general that would be better (see comment below)